src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -120,12 +147,64 @@ def _detect_performance_issue(data: Event, sdk_span: Any):
         DetectorType.N_PLUS_ONE_SPANS: NPlusOneSpanDetector(detection_settings, data),
     }
 
+    # Create performance issues for duplicate spans first
+    used_perf_issue_detectors = {
+        DetectorType.DUPLICATE_SPANS_HASH: DuplicateSpanHashDetector(detection_settings, data),
```

Comment:
We can just pick one of these (we probably want `hash`), and move forward with just the one to reduce confusion, it's not like we can't re-add the other back later, mostly since we are now using `hash` in the _group fingerprint_. Unless you were thinking to check out the output of these vs each other? 🤔

Reply:
that makes sense 👍 picking Hash one